packages/mermaid/src/docs/intro/syntax-reference.md

```diff
@@ -48,20 +48,20 @@ Configuration is the third part of Mermaid, after deployment and syntax. It deal
 
 If you are interested in altering and customizing your Mermaid Diagrams, you will find the methods and values available for [Configuration](../config/setup/README.md) here. It includes themes.
 This section will introduce the different methods of configuring the behaviors and appearances of Mermaid Diagrams.
-The following are the most commonly used methods, and they are all tied to Mermaid [Deployment](./n00b-gettingStarted.md) methods.
+The following are the most commonly used methods, and they are all tied to Mermaid [Deployment](./getting-started.md) methods.
 
 ### Configuration Section in the [Live Editor](https://mermaid.live).
 
 Here you can edit certain values to change the behavior and appearance of the diagram.
 
-### [The initialize() call](https://mermaid-js.github.io/mermaid/#/n00b-gettingStarted?id=_3-calling-the-javascript-api),
+### [The initialize() call](./getting-started#_3-calling-the-javascript-api)
```

Comment:
```suggestion
### [The initialize() call](./getting-started.md#_3-calling-the-javascript-api)
```

Very minor, but should this have a `.md`? extension? (Please ignore me if this doesn't work!).

Reply:
Across all the docs `.md` is the part of the links. So I decided to keep it consistent. This is not a link to a website anymore, which may be configured to derive data type by request headers. This is relative link to wherever the docs are hosted. I dare say, that this is done for GitHub compatibility, because [this link with md](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/intro/index.md) is working while [a link to the same file this without .md](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/intro/index) is not

Reply:
> Across all the docs .md is the part of the links. So I decided to keep it consistent.

I'm confused. So, shouldn't md be part of the link as @aloisklink suggested?

Reply:
Oh, sorry. I thought I added it, actually, my bad. It must be there for sure